startup/92-handler2.py

```diff
@@ -56,53 +56,3 @@ def shape(self):
 
     def close(self):
         self._handle.close()
-
-
-class EigerHandler2:
-    EIGER_MD_LAYOUT = {
-        'y_pixel_size': 'entry/instrument/detector/y_pixel_size',
-        'x_pixel_size': 'entry/instrument/detector/x_pixel_size',
-        'detector_distance': 'entry/instrument/detector/detector_distance',
-        'incident_wavelength': 'entry/instrument/beam/incident_wavelength',
-        'frame_time': 'entry/instrument/detector/frame_time',
-        'beam_center_x': 'entry/instrument/detector/beam_center_x',
-        'beam_center_y': 'entry/instrument/detector/beam_center_y',
-        'count_time': 'entry/instrument/detector/count_time',
-        'pixel_mask': 'entry/instrument/detector/detectorSpecific/pixel_mask',
-    }
-    specs = {'AD_EIGER2'}
-    def __init__(self, fpath, images_per_file):
-        # create pims handler
-        self._base_path = fpath
-        self._images_per_file = images_per_file
-
-    def __call__(self, seq_id):
-        master_path = '{}_{}_master.h5'.format(self._base_path, seq_id)
-        with h5py.File(master_path, 'r') as f:
-            md = {k: f[v].value for k, v in self.EIGER_MD_LAYOUT.items()}
-        # the pixel mask from the eiger contains:
-        # 1  -- gap
-        # 2  -- dead
-        # 4  -- under-responsive
-        # 8  -- over-responsive
-        # 16 -- noisy
-        pixel_mask = md['pixel_mask']
-        #pixel_mask[pixel_mask>0] = 1
-        #pixel_mask[pixel_mask==0] = 2
-        #pixel_mask[pixel_mask==1] = 0
-        #pixel_mask[pixel_mask==2] = 1
-        md['binary_mask'] = (md['pixel_mask'] == 0)
-        md['framerate'] = 1./md['frame_time']
-        # TODO Return a multi-dimensional PIMS seq.
-        return EigerImages2(master_path, self._images_per_file, md=md)
```

Comment:
This appears to be the only place `EigerImages2` is used, and `EigerHandler2` appears to never be used anymore.

If that's correct, maybe this whole file can be removed? Handlers will be provided automatically by `area-detector-handlers`.